Origin: Debian python2.7 2.7.18-8+deb11u1, debian/patches/site-locations.diff
Forwarded: not-needed
Description: Adapt Debian installation layout for the Tauthon namespace

# DP: Set site-packages/dist-packages

--- a/Lib/site.py
+++ b/Lib/site.py
@@ -19,6 +19,12 @@
 resulting directories, if they exist, are appended to sys.path, and
 also inspected for path configuration files.
 
+For Debian and derivatives, this sys.path is augmented with directories
+for packages distributed within the distribution. Local addons go
+into /usr/local/lib/tauthon<version>/dist-packages, Debian addons
+install into /usr/{lib,share}/tauthon<version>/dist-packages.
+/usr/lib/tauthon<version>/site-packages is not used.
+
 A path configuration file is a file whose name has the form
 <package>.pth; its contents are additional directories (one per line)
 to be added to sys.path.  Non-existing directories (or
@@ -288,10 +294,12 @@
         if sys.platform in ('os2emx', 'riscos'):
             sitepackages.append(os.path.join(prefix, "Lib", "site-packages"))
         elif os.sep == '/':
+            sitepackages.append(os.path.join(prefix, "local/lib",
+                                        "tauthon" + sys.version[:3],
+                                        "dist-packages"))
             sitepackages.append(os.path.join(prefix, "lib",
                                         "tauthon" + sys.version[:3],
-                                        "site-packages"))
-            sitepackages.append(os.path.join(prefix, "lib", "site-python"))
+                                        "dist-packages"))
         else:
             sitepackages.append(prefix)
             sitepackages.append(os.path.join(prefix, "lib", "site-packages"))
